_data/projectfooter.yaml

```diff
@@ -17,25 +21,31 @@ links:
         url: https://github.com/quarkusio
   - title: Get Help
     subfolderitems:
+      - page: Forums
+        url: https://groups.google.com/forum/#!forum/quarkus-dev
       - page: Chatroom
         url: https://quarkusio.zulipchat.com
-      - page: Google&nbsp;Groups
-        url: https://groups.google.com/forum/#!forum/quarkus-dev
-      - page: FAQ
-        url: /faq
 
 more_links:
-  - title: Quarkus is proudly made of a number of projects including ...
+  - title: Quarkus is made of community projects
     subfolderitems:
+      - page: Apache Camel
+        url: https://camel.apache.org
```

Comment:
I think Apache Camel should be at the bottom, especially with the extensions being outside of the tree now. I don't see the value in sorting that alphabetically, better have it by importance.